rfcs/2021-05-23-introduction.md

```diff
@@ -0,0 +1,41 @@
+# SMP agent introduction
+
+## Problem
+
+Allow an agent client to connect two connections it has directly, with the agent acting as an out-of-band channel.
+
+It can be used both separately as part of some client functionality, and as part of group protocol.
+
+## Solution
+
+A protocol with commands and message envelopes to exchange the information between parties to establish connection.
+
+### Commands and messages
+
+Below commands are for the scenario when A introduces B to M.
+
+- command `C:idAB INTRO C:idAM infoM` - introduce connection cIdB to connection cIdM (response is `C:idAB OK`)
```

Comment:
I'd suggest rephrasing since it's not introduction yet until B accepts. Also using Ids with different indexes in command and explanation (AB vs B) is confusing, just refer to parties - B, M?
`introduce connection cIdB to connection cIdM` -> `**proposal to** introduce B to M`

Reply:
I mean it doesn't really matter if these internal rfcs have ambiguities in formulations, but later they'll be added to protocol